packs/feats/regurgitate-mutagen.json

```
@@ -4,14 +4,14 @@
     "name": "Regurgitate Mutagen",
     "system": {
         "actionType": {
-            "value": "passive"
+            "value": "action"
         },
         "actions": {
-            "value": null
+            "value": 1
         },
         "category": "class",
         "description": {
-            "value": "<p><strong>Requirements</strong> You are under the effects of a mutagen.</p><hr /><p>You redirect a mutagen within your body to spit a stream of stomach acid at a foe. A creature within 30 feet takes @Damage[(floor(@actor.level/2))d6[acid]], with a @Check[reflex|against:alchemist|basic] save against your class DC. On a failure, the creature is also @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1} (or @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 2} on a critical failure). The mutagen's duration immediately ends.</p>"
+            "value": "<p><strong>Requirements</strong> You are under the effects of a mutagen.</p><hr /><p>You redirect a mutagen within your body to spit a stream of stomach acid at a foe. A creature within 30 feet takes @Damage[(floor(@actor.level/2))d6[acid]]{1d6 acid} damage for every 2 levels you have, with a @Check[reflex|against:alchemist|basic] save against your class DC. On a failure, the creature is also @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1} (or @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 2} on a critical failure). The mutagen's duration immediately ends.</p>"
```

```suggestion
            "value": "<p><strong>Requirements</strong> You are under the effects of a mutagen.</p><hr /><p>You redirect a mutagen within your body to spit a stream of stomach acid at a foe. A creature within 30 feet takes 1d6 acid damage for every 2 levels you have (@Damage[(floor(@actor.level/2))d6[acid]]), with a @Check[reflex|against:alchemist|basic] save against your class DC. On a failure, the creature is also @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 1} (or @UUID[Compendium.pf2e.conditionitems.Item.Sickened]{Sickened 2} on a critical failure). The mutagen's duration immediately ends.</p>"
```
How about this? Shows the scaled damage while keeping the text intact.